### DISCUSSION TITLE
"totals-information" calls two times on cart page

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#313] [1. Help]
Hi @vovayatsyuk 

The loader shows two times due to "totals-information" ajax request calling two times on the cart page.
please see here https://master-7rqtwti-vcte4ksdazidy.eu-4.magentosite.cloud/breeze/checkout/cart/

![Screenshot from 2024-08-07 20-07-39](https://github.com/user-attachments/assets/102b349b-f949-48d5-a6f3-812c1730e0b4)

It would be helpful if you check.

Thanks,
Pankaj

**Comment by pk-pankaj** [comment]
This only happens with Breeze, but it works fine with Luma.

**Comment by vovayatsyuk** [comment]
Thanks for reporting!

**Comment by vovayatsyuk** [comment]
Fixed: https://github.com/breezefront/module-breeze/commit/68af7d3319fe6a470674388be27f01bec8b54bbd
